config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -312,3 +312,55 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  - name: pull-perf-tests-clusterloader2-kubemark
+    always_run: true
+    skip_report: false
+    max_concurrency: 2
+    branches:
+      - master
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-e2e-kubemark-common: "true"
+    spec:
+      containers:
+        - args:
+            - --root=/go/src
+            - --repo=k8s.io/kubernetes=master
+            - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+            - --repo=k8s.io/release
+            - --upload=gs://kubernetes-jenkins/pr-logs
+            - --timeout=120
+            - --scenario=kubernetes_e2e
+            - --
+            - --build=bazel
+            - --cluster=
+            - --extract=local
+            - --gcp-master-size=n1-standard-2
+            - --gcp-node-image=gci
+            - --gcp-node-size=n1-standard-4
+            - --gcp-nodes=4
+            - --gcp-project=k8s-presubmit-scale
+            - --gcp-zone=us-east1-b
+            - --kubemark
+            - --kubemark-nodes=100
```

Comment:
Do we want 100? Why not 500 to be comparable with k/k?

Reply:
Done.